app/views/posts/show.html.erb

```diff
@@ -1,5 +1,16 @@
-<h1><%= @post.title %></h1>
+<div class="container">
+<div class="showpost">
 
+<div class="flex"><span><%= @post.id %># By : <%= @user.name %>  </span> <span> likes: X <%=  @post.commentsCounter  %> </span> <span> comments: Y <%=  @post.likesCounter %> </span></div>
+<p><%= @post.text %></p>
 
+</div>
+<div class="showpostcomment">
+<p>username: comment 1</p>
+<p>username: comment 2</p>
+<p>username: comment 3</p>
+<p>username: comment 4</p>
+</div>
```

Comment:
- [ ] [OPTIONAL] There is no need to do this because the comments should come from the database.